gemrb/core/CharAnimations.cpp

```diff
@@ -2079,13 +2079,13 @@ void CharAnimations::AddMHRSuffix(ResRef& dest, unsigned char StanceID,
 		case IE_ANI_CAST://looping
 			dest.Append("ca");
 			EquipData.Suffix = "ca";
-			Cycle = 8 + Orient;
+			Cycle = Orient;
 			break;
 
 		case IE_ANI_CONJURE://ending
 			dest.Append("ca");
 			EquipData.Suffix = "ca";
-			Cycle = Orient;
+			Cycle = 8 + Orient;
 			break;
```

Comment:
Which animation / creature did you test with?

Reply:
> Which animation / creature did you test with?

Animation of a humanoid (human). In the original, he spins his arms in circles and at the end pushes his arms forward. Before the edits, he first did pushes with his hands, and at the end he did a circle.